dokka-integration-tests/utilities/src/main/kotlin/org/jetbrains/dokka/it/systemVariableProviders.kt

```diff
@@ -1,4 +1,8 @@
-package org.jetbrains.dokka.dokkatoo.utils
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.it
```

Comment:
it looks like declarations from this file are not used now (at least IDEA shows no usages), is it for future or can be just removed?

Reply:
thanks, you're right it can be moved back.